### PR TITLE
[DRAFT] [DNM] adds aws-disconnected

### PR DESCRIPTION
**Pull request by wgahnagl**
open to have another alternative e2e test if wmco vsphere disconnected doesn't work

### CODE REVIEW

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

File: ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/post/windows-e2e-operator-test-aws-disconnected-post-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+chain:
+  as: windows-e2e-operator-test-aws-disconnected-post
+  steps:
+    - chain: upi-vsphere-windows-post
```
Comment (sebsoto) [inline ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/post/windows-e2e-operator-test-aws-disconnected-post-chain.yaml:4]: this shouldnt be vsphere

File: ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/pre/windows-e2e-operator-test-aws-disconnected-pre-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - ref: ipi-conf-aws
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest
+    - chain: ipi-install
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource-disconnected
+    - ref: windows-e2e-operator-test-mirror-images
+  documentation: >-
+    This chain provisions a OpenShift cluster with platform=none configuration
+    and a Windows instance in vSphere.
```
Comment (sebsoto) [inline ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/pre/windows-e2e-operator-test-aws-disconnected-pre-chain.yaml:19]: vSphere -> aws

**Review by sebsoto — CHANGES_REQUESTED** [review]
the mirror images step is failing

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
oh my god the cluster failed to launch with a timeout 

/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

getting some legitimately bizarre failures 
invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference) in the console operator, and Unable to connect to the server: dial tcp: lookup api.ci-op-v4gllnj4-66620.origin-ci-int-aws.dev.rhcloud.com on 172.30.0.10:53: no such host right after

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

the last failure was a documented bug, let's see if it's fixed this morning

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

it wasn't fixed! Maybe now?

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

why is this test so unstable, this is also a flake 
this is somehow more unstable than vsphere, which isn't even supposed to run as a CI job 

```
Created stack

Waiter StackCreateComplete failed: Waiter encountered a terminal failure state: For expression "Stacks[].StackStatus" we matched expected path: "ROLLBACK_COMPLETE" at least once
{"component":"entrypoint","error":"wrapped process failed: exit status 255","file":"sigs.k8s.io/prow/pkg/entrypoint/run.go:84","func":"sigs.k8s.io/prow/pkg/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2024-05-14T14:33:56Z"}
error: failed to execute wrapped command: exit status 255
```

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

test didn't flake the last time! 
Testing with a much cleaner version

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

didn't trigger argh

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator 

another flake :(

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-disconnected-e2e-operator

**Pull request closed without merging**